utils/lib/connection_database.py

```diff
@@ -25,10 +25,11 @@ def create_tables(conn):
     c.execute(
         """
 INSERT INTO
-    switch(name, internal_capacitance, drive_resistance, intrinsic_delay, switch_type)
+    switch(name, internal_capacitance, drive_resistance, intrinsic_delay, penalty_cost, switch_type)
 VALUES
-    ("__vpr_delayless_switch__", 0.0, 0.0, 0.0, "mux"),
-    ("short", 0.0, 0.0, 0.0, "short")
+    ("__vpr_delayless_switch__", 0.0, 0.0, 0.0, 0.0, "mux"),
+    ("__vpr_penalty_switch__", 0.0, 0.0, 0.0, 1.0, "mux"),
```

Comment:
If you follow the comment [here](https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1267/files#r370238209), then adding `__vpr_penalty_switch__` will no longer be required.

Reply:
Done